pve-mod-gui-sensors.sh

```diff
@@ -633,13 +659,35 @@ Ext.define('PVE.mod.TempHelper', {\n\
 			a\
 			\\
 	{\n\
+		itemId: 'separatorBox',\n\
 		xtype: 'box',\n\
 		colspan: 2,\n\
 		padding: '0 0 20 0',\n\
 	},
 		}" "$pvemanagerlibjs"
 		fi
 
+		if [[ $enableSystemInfo == "true" ]]; then
+			sed -i "/^Ext.define('PVE.node.StatusView',/ {
+				:a;
+				/title:/!{N;ba;}
+				:b;
+				/'separatorBox.*},/!{N;bb;}
```

Comment:
Currently dependent on separator box. Should be before the block, so sed should be updated:





	{
	    colspan: 2,
	    title: gettext('Kernel Version'),
	    printBar: false,
	    // TODO: remove with next major and only use newish current-kernel textfield
	    multiField: true,
	    //textField: 'current-kernel',
	    renderer: ({ data }) => {
		if (!data['current-kernel']) {
		    return data.kversion;
		}
		let kernel = data['current-kernel'];
		let buildDate = kernel.version.match(/\((.+)\)\s*$/)?.[1] ?? 'unknown';
		return `${kernel.sysname} ${kernel.release} (${buildDate})`;
	    },
	    value: '',
	},